client/src/views/files/FileDetailsModal.vue

```diff
@@ -94,7 +94,10 @@
           </ion-label>
           <div class="file-info-path-value">
             <ion-text class="file-info-path-value__text body">
-              {{ shortenFileName(entry.path, { maxLength: 60, prefixLength: 20, suffixLength: 30 }) }}
+              <span v-if="toggleShowFullPath">
```

Comment:
Logic is reversed, cropped path should be shown by default and full path should be shown after clicking